spanner/tests/unit/test_database.py

```diff
@@ -271,6 +274,7 @@ def with_scopes(self, scopes):
         expected_scopes = (SPANNER_DATA_SCOPE,)
         client = _Client()
         client_info = client._client_info = mock.Mock()
+        client_options = client._client_options = mock.Mock()
```

Comment:
Currently the `client_options` variable isn't being used and is causing the lint check to fail. Did you mean to use it in an assert similar to the previous test?